src/main/java/macrobase/diagnostics/ScoreDumper.java

```diff
@@ -0,0 +1,41 @@
+package macrobase.diagnostics;
```

Comment:
yes actually. I had some code that would run diagnostics not from tests. I still want to, because sometimes all i need is run macrobase on a given dataset and dump the results in a file to be able to plot. Currently my solution is to write a test for that case, which does not give me freedom to automatically iterate over configuration parameters or datasets. So I think there should be a diagnostics pipeline or switch in the main package, I can't decide which one would be nicer and more maintainable.